includes/Checker/Check_Runner.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Interface WordPress\Plugin_Check\Checker\Check_Runner
+ *
+ * @package plugin-check
+ */
+
+namespace WordPress\Plugin_Check\Checker;
+
+/**
+ * Interface for runner classes like AJAX runner or CLI runner.
+ *
+ * @since n.e.x.t
+ */
+interface Check_Runner {
+
+	/**
+	 * Determine if the current request is intended for the plugin checker.
```

Comment:
Function/method descriptions should start with 3rd person verb.
```suggestion
	 * Determines if the current request is intended for the plugin checker.
```